.github/workflows/build-native.yml

```diff
@@ -2,6 +2,22 @@ name: Build Native
 
 on:
   workflow_dispatch:
+  push:
+    branches:
+      - master
```

Comment:
Let's remove this condition as directly modifying the master branch is not a good practice.